clang/docs/ReleaseNotes.rst

```diff
@@ -1246,6 +1323,9 @@ Moved checkers
   checker ``alpha.security.ArrayBound`` (which was searching for the same kind
   of bugs with an different, simpler and less accurate algorithm) is removed.
```

Comment:
```suggestion
  of bugs with a different, simpler and less accurate algorithm) is removed.
```

Reply:
I decided to drop the whole second part of the sentence. I don't think it adds much value. If anything, only confuses people.
fff9d58a1edde25edfcae07abec17da708a3d2a7